rules-tests/DeadCode/Rector/ClassMethod/RemoveUnusedPrivateMethodRector/Fixture/skip_magic_call.php.inc

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Rector\Tests\DeadCode\Rector\ClassMethod\RemoveUnusedPrivateMethodRector\Fixture;
+
+/**
+ * @see https://3v4l.org/Lh5O5
+ */
+class SkipMagicCall
+{
+    private function test($params)
```

Comment:
Why should „test“ not be removed.. its still unused?

Reply:
I've updated fixture test case to proof it https://github.com/rectorphp/rector/pull/5932/commits/bafe6d097c890ffa2c7f92444b9f8f709936de91 . It called with `$this->__call('test', []);`